src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        String[] sampleNames = {"John", "Andrew", "Kate"};
+        String[] scriptArray = {
+                "25.04.2019 John 60 50",
+                "25.04.2019 Andrew 3 200",
```

Comment:
You can have Main file locally, but it shouldn't be committed, remove it from PR